### ISSUE TITLE
TD widths ignored

### ISSUE THREAD

**Issue by fsodergren** [Xportability/css-to-pdf#24]
This solution will work perfect for me, though for some reason the  export is not respecting TD widths. 
http://codepen.io/fsodergren/pen/dMmdBQ
It sets TD widths to 50% instead of what's in the style. Any idea on how to fix this?

**Comment by cbrophy78** [comment]
Not sure if you figured this out, but what I noticed is:
1. No matter what I put for the width of the `<TD>` in the print style it did not effect the output.
2. However when I put the width on a `<TH>` it governed the whole column.
3. Some things still ended up wonky and was able to still effect the [foStyle](http://www.cloudformatter.com/CSS2Pdf.CustomTipsTricks.XSLFOStyling) on the text inside the `<TH>`.

If there is a better solution in sizing things I'd like to see it. I wasn't able to get a viable PDF based on my print css to match a printed sheet in div form. I needed to reformat everything into tables. Luckily the project was a static layout rather than responsive.
